lib/modules/platform/azure/util.ts

```diff
@@ -86,6 +86,25 @@ const stateMap = {
   [PullRequestStatus.Completed]: 'merged',
 } as Record<PullRequestStatus, PrState | undefined>;
 
+export function getRenovatePrFormatForCachedPr(
+  cachedPr: AzurePr,
+  azurePr: GitPullRequest,
+): AzurePr {
+  cachedPr.state = stateMap[azurePr.status!] ?? 'open';
+  if (azurePr.title) {
+    cachedPr.title = azurePr.title;
+  }
+  if (azurePr.description) {
+    cachedPr.bodyStruct = getPrBodyStruct(azurePr.description);
+  }
+  if (azurePr.targetRefName) {
+    cachedPr.targetBranch = getBranchNameWithoutRefsheadsPrefix(
+      azurePr.targetRefName,
+    );
+  }
+  return cachedPr;
```

Comment:
What's the point of returning a modified `cachedPr` if you are mutating the input object?